api-description/forms/pk/post.json

```diff
@@ -2,22 +2,36 @@
     "description": "Endpoint to answer a form",
     "url": "http://localhost:8000/forms/1",
     "fields": {
-        "answers": {
-            "1": "Friends",
-            "2": "one_time",
-            "3": ["two_times", "three_times"],
-            "4": 5
-        }
+        "survey": 1,
```

Comment:
It's a bit strange that I have to do that though, because it's the `pk` right? So if I'm sending it to url with a `pk` why do I also have to send the `pk` ?